src/modality-specific-files/behavioral-experiments.md

```diff
@@ -38,7 +38,7 @@ The definitions of these fields can be found in
 and a guide for using macros can be found at
  https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
 -->
-{{ MACROS___make_columns_table("task.Behavioral") }}
+{{ MACROS___make_columns_table("events.Behavioral") }}
```

Comment:
On line 31 above, it mentions `_beh.tsv`.  Should it also mention `_beh.json`?

Some comment about having a space between / and stimuli in some places resulting in it being split up on separate lines.